transforms/language/html2parquet/notebooks/html2parquet.ipynb

```diff
@@ -0,0 +1,220 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c4f9c952-cb3b-40f1-bfb5-00d9a43a5715",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install data-prep-toolkit==0.2.2.dev2\n",
+    "!pip install 'data-prep-toolkit-transforms[html2parquet]==0.2.2.dev2'\n",
+    "!pip install pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "20663a67-5aa1-4b61-b989-94201613e41f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from data_processing.runtime.pure_python import PythonTransformLauncher\n",
+    "from data_processing.utils import ParamsUtils\n",
+    "\n",
+    "from html2parquet_transform_python import Html2ParquetPythonTransformConfiguration\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e75f6922-eb0f-4164-a536-f96393e04604",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ast\n",
+    "\n",
+    "# create parameters\n",
+    "local_conf = {\n",
+    "    \"input_folder\": \"/path/to/your/input/folder\",\n",
+    "    \"output_folder\": \"/path/to/your/output/folder\",\n",
+    "}\n",
+    "\n",
+    "params = {\n",
+    "    # Data access. Only required parameters are specified\n",
+    "    \"data_local_config\": ParamsUtils.convert_to_ast(local_conf),\n",
+    "    \"data_files_to_use\": ast.literal_eval(\"['.zip', '.html']\"),\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "4d2354db-1bb3-4a71-98df-f0f148af3a02",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "17:09:40 INFO - html2parquet parameters are : {'output_format': <html2parquet_output_format.MARKDOWN: 'markdown'>, 'favor_precision': <html2parquet_favor_precision.TRUE: 'True'>, 'favor_recall': <html2parquet_favor_recall.TRUE: 'True'>}\n",
+      "17:09:40 INFO - pipeline id pipeline_id\n",
+      "17:09:40 INFO - code location None\n",
+      "17:09:40 INFO - data factory data_ is using local data access: input_folder - input output_folder - output\n",
+      "17:09:40 INFO - data factory data_ max_files -1, n_sample -1\n",
+      "17:09:40 INFO - data factory data_ Not using data sets, checkpointing False, max files -1, random samples -1, files to use ['.html'], files to checkpoint ['.parquet']\n",
+      "17:09:40 INFO - orchestrator html2parquet started at 2024-11-13 17:09:40\n",
+      "17:09:40 INFO - Number of files is 1, source profile {'max_file_size': 0.2035503387451172, 'min_file_size': 0.2035503387451172, 'total_file_size': 0.2035503387451172}\n",
+      "17:09:47 INFO - Completed 1 files (100.0%) in 0.111 min\n",
+      "17:09:47 INFO - Done processing 1 files, waiting for flush() completion.\n",
+      "17:09:47 INFO - done flushing in 0.0 sec\n",
+      "17:09:47 INFO - Completed execution in 0.111 min, execution result 0\n"
+     ]
+    }
+   ],
```

Comment:
This is the old style of Notebook, before restructuring in PR #809 is merged. In that PR, it is V0 in its name. After 809 is merged, we want the  Notebook to be the html2parquet.ipynb in that PR.

Reply:
I reviewed PR #809 and now I understand your point. My understanding was that we keep both the old and new layouts, so this PR may be relevant to the old layout. I might need to update the README for the new layout later. @touma-I, please correct me if I'm wrong.

Reply:
@sungeunan-ibm @shahrokhDaijavad sorry for causing all this confusion. The notebook we have right now in this PR is the right one. The new format will have to wait until we merge #809 sometime in early December. That said, is it possible to add a few markdown cells explaining what the cell below does?